### PR TITLE
icons archive generation

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

File: ver/us/symbol_addrs.txt [hunk 1]
```diff
@@ -2362,8 +2362,8 @@ IES_ITEM_137 = 0x8008DDB4; // rom:0x691B4
 gItemEntityScripts = 0x8008DDD4; // rom:0x691D4
 gItemIconRasterOffsets = 0x8008E3D8; // rom:0x697D8
 gItemIconPaletteOffsets = 0x8008E94C; // rom:0x69D4C
-D_8008EEC0 = 0x8008EEC0; // rom:0x6A2C0
-D_8008EEF0 = 0x8008EEF0; // rom:0x6A2F0
+PartnerMenuIndex_to_ID = 0x8008EEC0; // rom:0x6A2C0
```
Comment (ethteck) [inline ver/us/symbol_addrs.txt:2365]: I'm not a big fan of the underscores in the names. Could we do `PartnerMenuIndexToID` and `PartnerIDToMenuIndex` ?

Comment (nanaian) [reply ver/us/symbol_addrs.txt:2365]: PartnerMenuIndexToId or PartnerMenuIndexIdMap I'd say

**Comment by BowserSlug** [comment]
src/C50A0.c:15: warning: redundant redeclaration of `icon_ROM_START' in same scope
ver/ique/build/include/ld_addrs.h:3588: warning: previous declaration of `icon_ROM_START'
src/hud_element.c:8: warning: redundant redeclaration of `icon_ROM_START' in same scope
ver/ique/build/include/ld_addrs.h:3588: warning: previous declaration of `icon_ROM_START'

**Review by ethteck — APPROVED** [review]

File: src/415D90.c [hunk 2]
```diff
@@ -3460,7 +3460,7 @@ void btl_state_update_player_menu(void) {
         case BTL_SUBSTATE_PLAYER_MENU_CHANGE_MEMBER_1:
             entryIdx = 0;
             for (i = 1; i < ARRAY_COUNT(playerData->partners); i++) {
-                s32 partnerId = D_8008EEC0[i];
+                s32 partnerId = PartnerIDFromMenuIndex[i];
```
Comment (nanaian) [inline src/415D90.c:3463]: I'm still not sure about the capital "D" in ID. See https://stackoverflow.com/questions/15526107/acronyms-in-camelcase for discussion

**Pull request merged**